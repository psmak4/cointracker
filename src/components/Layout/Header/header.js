import { Container, Navbar } from 'react-bootstrap'

const Header = () => {
	return (
		<Navbar bg='dark' variant='dark' as='header'>
			<Container fluid className='px-4'>
				<Navbar.Brand>Cointracker</Navbar.Brand>
			</Container>
		</Navbar>
	)
}

export default Header
