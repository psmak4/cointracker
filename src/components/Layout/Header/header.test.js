import { render, screen } from '@testing-library/react'
import Header from './header'

describe('<Header />', () => {
	test('renders correctly', async () => {
		const { container } = render(<Header />)

		expect(container).toMatchSnapshot()

		const text = await screen.findByText('Cointracker')
		expect(text).toBeInTheDocument()
	})
})
