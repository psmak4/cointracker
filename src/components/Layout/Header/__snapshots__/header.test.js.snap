// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<div>
  <header
    class="navbar navbar-expand navbar-dark bg-dark"
    role="navigation"
  >
    <div
      class="px-4 container-fluid"
    >
      <span
        class="navbar-brand"
      >
        Cointracker
      </span>
    </div>
  </header>
</div>
`;
